src/vos/evtree.c

```diff
@@ -3044,6 +3052,10 @@ evt_desc_csum_fill(struct evt_context *tcx, struct evt_desc *desc,
 			csum->cs_buf_len, csum_buf_len);
 	} else {
 		memcpy(desc->pt_csum, csum->cs_csum, csum_buf_len);
+		if (csum_bufp != NULL) {
+			D_ASSERT(csum->cs_csum == *csum_bufp);
```

Comment:
is there some hard requirement that the buffer passed in must be set to this value?    If so, that should be documented in the interface.   But I don't see why one can't pass in some uninitialized pointer.  If the assert is checking something in the aggregation logic, this is a weird place to put it.

Reply:
I took the assert out.